.github/workflows/check-push.yml

```diff
@@ -0,0 +1,36 @@
+name: Develop Workflow with .Net
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+permissions: read-all      
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      
+      - name: Build using .Net
+        run: dotnet build
+          
```

Comment:
Also, I recommend you to write a job for the branches which will go back to develop, like the one which is on `check-pull-request.yml` file

Reply:
Done